tests/adiabatic_conditions_composition_function/depth_average.gnuplot

```diff
@@ -4,74 +4,74 @@
 #
 # For a description of the GNUPLOT format see the GNUPLOT manual.
 #
-# <depth> <time> <temperature> <C_0> <adiabatic_temperature> <adiabatic_pressure> <adiabatic_density> <adiabatic_density_derivative> <velocity_magnitude> <sinking_velocity> <viscosity> <vertical_heat_flux> <vertical_mass_flux> 
-0 0 1613 0 1613 0 3300 0 7.06667e-16 7.80426e-17 1e+21 3.92932e-08 4.95593e-13 
-289000 0 1613 0 1613 0 3300 0 7.06667e-16 7.80426e-17 1e+21 3.92932e-08 4.95593e-13 
+# <depth> <time> <temperature> <C_1> <adiabatic_temperature> <adiabatic_pressure> <adiabatic_density> <adiabatic_density_derivative> <velocity_magnitude> <sinking_velocity> <viscosity> <vertical_heat_flux> <vertical_mass_flux> <C_1_mass> 
```

Comment:
This is curious. Do we start our compositional field names with 1 internally? If so this new output is actually a fix.

Reply:
Yes, this is correct. We start with C_1 for the names, so this is a fix.